oresat_configs/scripts/gen_xtce.py

```diff
@@ -144,7 +144,26 @@ def write_xtce(config: OreSatConfig, dir_path: str = ".") -> None:
     tm_meta = ET.SubElement(root, "TelemetryMetaData")
     tm_meta_para = ET.SubElement(tm_meta, "ParameterTypeSet")
 
-    # Hard-code the 128b type for the AX.25 parameter
+    # hard-code the uint32 time type for the crc32
```

Comment:
Can remove the word `time` here. Looks like it was a copy/paste from `unix_time`?

Reply:
Fixed